docs/users.rst

```diff
@@ -113,6 +113,27 @@ The simplest, smallest, and fastest is the beamville scenario (described below).
 
   ./gradlew :run -PappArgs="['--config', 'test/input/beamville/beam.conf']"
 
+
+The BEAM application by default sets max RAM allocation to 140g (see **maxRAM** setting in gradle.properties). This needs to
+be adjusted based on the available memory on your system.
+
+The max allocatable RAM value can be overriden by setting the environment variable **MAXRAM** to the required value.
+
+On Ubuntu , the environment variable can be set using the below command :
+
+.. code-block:: bash
+
+    export MAXRAM=10g
+
+where 10g = 10GB
+
+Similarly on windows it can be set using the below command :
+
+.. code-block:: bash
+
+    setx MAXRAM="10g"
```

Comment:
Just an FYI that setx will set the variable permanently (different than export above). Should we say set for consistency? @colinsheppard @wrashid